src/PageTabs.tsx

```diff
@@ -276,6 +288,7 @@ function getBlockUUID(element: HTMLElement) {
 function stop(e: Event) {
   e.stopPropagation();
   e.stopImmediatePropagation();
+  e.preventDefault(); // prevent scrolling from middle mouse button click
```

Comment:
does it also stop scrolling with wheel event?

Reply:
what do you mean? If it is also stopping the normal scroll wheel event (not from the middle mouse button)?
The stop method is only called on middle mouse button click or on Ctrl + left mouse button click.